### PR TITLE
Add 1.4.6 release blog post

### PR DESCRIPTION
**Pull request by deathbeam**
Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+that when clicked will pin the config option to top)
+- Search bar in configuration panel will now always stay on top
+- Each configuration page for plugins now have back and on/off buttons at the top of panel
+
+![configbackbutton](/img/blog/1.4.6-Release/configbackbutton.png)
+![configfavorites](/img/blog/1.4.6-Release/configfavorites.png)
+![configtags](/img/blog/1.4.6-Release/configtags.png)
+![configtooltip](/img/blog/1.4.6-Release/configtooltip.png)
+
+Thanks to another awesome contribution from [PandahRS](https://github.com/PandahRS) who spent
```
Comment (jplsek) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: @

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+---
+
+A lightbox solver plugin was added which tells you the pattern to solve lightbox only
+after few clicks (that are needed to figure out the pattern). Contributed by
+[@Adam-](https://github.com/Adam-).
+
+![lightboxsolver](/img/blog/1.4.6-Release/lightboxsolver.png)
+
+Helper for Varrock Museum Quiz was added as well that highlights correct answer to
+questions in Varrock Musuem basement. Thanks to [vikke1234](https://github.com/vikke1234)
```
Comment (jplsek) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: @

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+who contributed this feature.
+
+![varrockmuseumsolver](/img/blog/1.4.6-Release/varrockmuseumsolver.png)
+
+The configuration panel was significantly improved thanks to big contribution from
+[takuyakanbr](https://github.com/takuyakanbr). Some of the improvements are:
```
Comment (jplsek) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: @

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+![discord1](/img/blog/1.4.6-Release/discord1.png)
+![discord2](/img/blog/1.4.6-Release/discord2.png)
+
+Among the main highlighted features there has been several other improvements and bugfixes this release:
+
+ * !total, !clues and !lvl commands now properly show ranks based on account types
+ * plenty of new food types has been added to Item Stats plugin
+ * SOTD timer now properly works also for SOTL
+ * obstacles for Agility Pyramid entrance and Morytania pirate ship has been added to Agility plugin
```
Comment (jplsek) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: Capitalize first letter in sentence

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+Due to privacy reasons, these new location-based features are disabled in any PVP scenario, such
+as PVP worlds, High Risk worlds, DMM and Seasonal DMM worlds.
+
+![discord1](/img/blog/1.4.6-Release/discord1.png)
+![discord2](/img/blog/1.4.6-Release/discord2.png)
+
+Among the main highlighted features there has been several other improvements and bugfixes this release:
+
+ * !total, !clues and !lvl commands now properly show ranks based on account types
+ * plenty of new food types has been added to Item Stats plugin
```
Comment (jplsek) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: Capitalize first letter in sentence

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+author: Tomas
+---
+
+A lightbox solver plugin was added which tells you the pattern to solve lightboxs after
+observing a few of the switches. Contributed by [@Adam-](https://github.com/Adam-).
+
+![lightboxsolver](/img/blog/1.4.6-Release/lightboxsolver.png)
+
+A helper for Varrock Museum Quiz was added as well that highlights correct answer to
+questions in Varrock Musuem basement. Thanks to [vikke1234](https://github.com/vikke1234)
```
Comment (Nightfirecat) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: `s/Musuem/Museum/`

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+a lot of new features, including display of **city** you are in, **dungeon** you are currently
+exploring, **boss** you are currently fighting and even **minigame** you are currently playing.
+
+Due to privacy reasons, these new location-based features are disabled in any PVP scenario, such
+as PVP worlds, High Risk worlds, DMM and Seasonal DMM worlds.
+
+![discord1](/img/blog/1.4.6-Release/discord1.png)
+![discord2](/img/blog/1.4.6-Release/discord2.png)
+
+Among the main highlighted features there has been several other improvements and bugfixes this release:
```
Comment (Nightfirecat) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: Among seems like the wrong word for this... Also `s/has/have/`.

The following might be a better way to word this sentence.

```
In addition to the highlighted features, there have been several other improvements and bugfixes this release:
```

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md (hunk 4 above)
Comment (Nightfirecat) [reply src/_posts/2018-07-05-07-00-1.4.6-Release.md]: `s/has/have/`

File: src/_posts/2018-07-05-07-00-1.4.6-Release.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
+---
+title: '1.4.6 Release'
+description: 'Lightbox and Varrock Museum quiz solver, config panel tags and favorites, Discord bosses & minigames'
+author: Tomas
+---
+
+A lightbox solver plugin was added which tells you the pattern to solve lightboxs after
```
Comment (Nightfirecat) [inline src/_posts/2018-07-05-07-00-1.4.6-Release.md]: `s/lightboxs/lightboxes/`

**Pull request merged**